client/js/jsonedit.js

```diff
@@ -818,7 +864,12 @@ function jeShowCommands() {
           if(!command.currentKey && !usedKeys[key])
             command.currentKey = key;
         usedKeys[command.currentKey] = true;
-        commandText += `Ctrl-${command.currentKey}: ${name.replace(command.currentKey, '<b>' + command.currentKey + '</b>')}\n`;
+        var displayKey = command.currentKey;
+        if (displayKey == 'ArrowUp')
+          displayKey = '⬆';
+        if (displayKey == 'ArrowDown')
+          displayKey = '⬇';
```

Comment:
This is fine for now. If there are more, this should be more like `let displayKey = { ArrowUp: '⬆', ArrowDown: '⬇'}[command.currentKey] || command.currentKey;` (not necessarily on one line).

Reply:
Yes, good point. I actually added a third arrow key for the "convert to applyVariables" and used your suggestion